blog/r-tidyfinance-0-4-0-new-data-downloads/index.qmd

```diff
@@ -0,0 +1,126 @@
+---
+title: "tidyfinance 0.4.0: new data downloads"
+author:
+  - name: Christoph Scheuch
+    url: https://www.linkedin.com/in/christophscheuch/
+    affiliations:
+      - name: Independent
+  - name: Stefan Voigt
+    url: https://www.voigtstefan.me/
+    affiliations:
+      - name: University of Copenhagen
+      - name: Danish Finance Institute
+  - name: Patrick Weiss
+    url: https://sites.google.com/view/patrick-weiss
+    affiliations:
+      - name: Reykjavik University
+      - name: WU Vienna University of Economics and Business
+date: "2024-08-30"
+description: tidyfinance 0.4.0 is now on CRAN. Discover the new data download options it includes. 
+image: thumbnail.png
+image-alt: Abstract illustration of a data flow pipeline with arrows and lines moving from financial icons like stock charts, currency symbols, and graphs towards a central tidy data frame. The background features a gradient of deep blues and greens, symbolizing finance and technology. The text ‘tidyfinance 0.4.0’ and a small CRAN logo are prominently displayed, with additional subtle financial icons integrated into the design. Created with DALL-E.
+categories: 
+  - Data
+  - R
+--- 
+
+We are happy to annouce the release of version 0.4.0 of the `tidyfinance` R package [on CRAN](https://CRAN.R-project.org/package=tidyfinance). The package contains a set of helper functions for empirical research in financial economics, addressing a variety of topics covered in [Tidy Finance with R](../../r/index.qmd) (TFWR). We designed the package to provide easy shortcuts for the applications that we discuss in the book. If you want to inspect the details of the package or propose new features, feel free to visit the package repository on [Github](https://github.com/tidy-finance/r-tidyfinance).
+
+As the new release brings many new features, we split them up into two blog posts. In this blog post, we discuss the new data downloads that `tidyfinance` now supports. 
+
+## Install the package
+
+You can install the released version of `tidyfinance` from CRAN via:
+
+```{r}
+#| eval: false
+install.packages("tidyfinance")
+```
+
+You can install the development version of tidyfinance from GitHub using the `pak` package:
+
+```{r}
+#| eval: false
+pak::pak("tidy-finance/r-tidyfinance")
+```
+
+You then load the package via:
+
+```{r}
+library(tidyfinance)
+```
+
+## Download stock data
+
+You can download stock prices from [Yahoo Finance](https://finance.yahoo.com/) using the type `stock_prices` and provide symbols:
+
+```{r}
+download_data("stock_prices", symbols = c("AAPL", "MSFT"))
+```
+
+As you can see, we included defaults for `start_date` and `end_date` that you can of course overwrite:
```

Comment:
I think I would mention that we generally introduced such defaults across all datasets to allow for fast initial data analysis. In the tidyfinance package, we consistently provide the arguments start_date and end_date for your convenience.